pkg/daemon/metrics.go

```diff
@@ -95,13 +95,13 @@ func updatePTPMetrics(process, iface string, offsetFromMaster, maxOffsetFromMast
 }
 
 // extractMetrics ...
-func extractMetrics(processName, iface, output string) {
+func extractMetrics(processName string, iface []string, output string) {
 	if strings.Contains(output, " max ") {
 		offsetFromMaster, maxOffsetFromMaster, frequencyAdjustment, delayFromMaster := extractSummaryMetrics(processName, output)
-		updatePTPMetrics(processName, iface, offsetFromMaster, maxOffsetFromMaster, frequencyAdjustment, delayFromMaster)
+		updatePTPMetrics(processName, iface[0], offsetFromMaster, maxOffsetFromMaster, frequencyAdjustment, delayFromMaster)
```

Comment:
why you put here only the first interface?
I think you need to change this to support multiple interfaces metrics

/cc @aneeshkp

Reply:
I didn't want to change the signature of that function here.  I expect Aneesh will update this when he changes updatePTPMetrics.